Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-flask = "*"
+flask = "==0.12.3"
```

Comment:
Any reason this is pinned?

Reply:
yes, later versions don't play nicely with the pinned werkzeug = "<0.15" . tried playing around with this and it all gets messy.  i.e.  wants <0.15 & > 0.15 at the same time

Reply:
@LukeLoze1 What happens when you unpin (-> `"*"`) both?

Reply:
Ah, never mind. Tests fail.